tests/test_deep/test_agents/test_cem.py

```diff
@@ -0,0 +1,10 @@
+from genrl.agents import CEM
+from genrl.environments import VectorEnv
+from genrl.trainers import OnPolicyTrainer
+
+
+def test_CEM():
+    env = VectorEnv("CartPole-v0", 1)
```

Comment:
Why set it to 1? It does work with multiple envs right?

Reply:
Yeah it does